include/z64scene.h

```diff
@@ -417,4 +419,139 @@ typedef enum {
+    /* 0x0F */ SCENECMD_ID_LIGHT_LIST,
+    /* 0x10 */ SCENECMD_ID_TIME_SETTINGS,
+    /* 0x11 */ SCENECMD_ID_SKYBOX_SETTINGS,
+    /* 0x12 */ SCENECMD_ID_SKYBOX_DISABLES,
+    /* 0x13 */ SCENECMD_ID_EXIT_LIST,
+    /* 0x14 */ SCENECMD_ID_END,
+    /* 0x15 */ SCENECMD_ID_SOUND_SETTINGS,
+    /* 0x16 */ SCENECMD_ID_ECHO_SETTINGS,
+    /* 0x17 */ SCENECMD_ID_CUTSCENE_LIST,
+    /* 0x18 */ SCENECMD_ID_ALTERNATE_HEADER_LIST,
+    /* 0x19 */ SCENECMD_ID_MISC_SETTINGS,
+    /* 0x1A */ SCENECMD_ID_TEXTURE_ANIM_LIST,
+    /* 0x1B */ SCENECMD_ID_ACTOR_CUTSCENE_LIST,
+    /* 0x1C */ SCENECMD_ID_MINIMAP_INFO,
+    /* 0x1D */ SCENECMD_ID_UNUSED_1D,
+    /* 0x1E */ SCENECMD_ID_MINIMAP_COMPASS_ICON_INFO
+} SceneCommandTypeID;
+
+#define SCENECMD_SPAWN_LIST(spawnNum, spawnList)                 \
+    { SCENECMD_ID_SPAWN_LIST, spawnNum, CMD_PTR(spawnList) }
+
+#define SCENECMD_ACTOR_LIST(actorNum, actorList)                 \
+    { SCENECMD_ID_ACTOR_LIST, actorNum, CMD_PTR(actorList) }
+
+#define SCENECMD_CAM_LIST(camNum, camList)       \
+    { SCENECMD_ID_ACTOR_CUTSCENE_CAM_LIST, camNum, CMD_PTR(camList) }
+
+#define SCENECMD_COL_HEADER(colHeader)                    \
+    { SCENECMD_ID_COL_HEADER, 0, CMD_PTR(colHeader) }
+
+#define SCENECMD_ROOM_LIST(roomNum, roomList)                 \
+    { SCENECMD_ID_ROOM_LIST, roomNum, CMD_PTR(roomList) }
+
+#define SCENECMD_WIND_SETTINGS(xDir, yDir, zDir, strength)                     \
+    { SCENECMD_ID_WIND_SETTINGS, 0, CMD_BBBB(xDir, yDir, zDir, strength) }
+
+#define SCENECMD_ENTRANCE_LIST(entranceList)                    \
+    { SCENECMD_ID_ENTRANCE_LIST, 0, CMD_PTR(entranceList) }
+
+#define SCENECMD_SPECIAL_FILES(elfMessageFile, specialObject)               \
+    { SCENECMD_ID_SPECIAL_FILES, elfMessageFile, CMD_W(specialObject) }
+
+#define SCENECMD_ROOM_BEHAVIOR(currRoomUnk3, currRoomUnk2, currRoomUnk5, msgCtxunk12044, enablePosLights,  \
+                                kankyoContextUnkE2)                                                         \
+    {                                                                                                       \
+        SCENECMD_ID_ROOM_BEHAVIOR, currRoomUnk3,                                                           \
+            currRoomUnk2 | _SHIFTL(currRoomUnk5, 8, 1) | _SHIFTL(msgCtxunk12044, 10, 1) |                   \
+                _SHIFTL(enablePosLights, 11, 1) | _SHIFTL(kankyoContextUnkE2, 12, 1)                        \
+    }
+
+#define SCENECMD_UNUSED_9()                   \
+    { SCENECMD_ID_UNUSED_9, 0, CMD_W(0) }
+
+#define SCENECMD_MESH(meshHeader)                    \
+    { SCENECMD_ID_MESH, 0, CMD_PTR(meshHeader) }
+
+#define SCENECMD_OBJECT_LIST(objectNum, objectList)                 \
+    { SCENECMD_ID_OBJECT_LIST, objectNum, CMD_PTR(objectList) }
+
+#define SCENECMD_POS_LIGHT_LIST(unusedLightNum, unusedLightList)             \
+    { SCENECMD_ID_POS_LIGHT_LIST, unusedLightNum, CMD_PTR(unusedLightList) } 
+
```

Comment:
We talked about it in discord but we can change this to just `LIGHT_LIST`, remove the word POS

Reply:
additionally i would take `unused` out of the arg names.
And instead of lightNum it should prob be `numLights`. this goes for a few other things in this file